.github/workflows/automatic-doc-checks.yml

```diff
@@ -0,0 +1,23 @@
+name: Main Documentation Checks
+
+on:
+  push:
```

Comment:
This should be made period, or disabled, similar to the one on main. See: https://github.com/juju/juju/pull/18427